Sources/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -110,9 +110,17 @@ extension Graph where Value == Int {
         self.init([source, sink] + internalNodes)
         for node in internalNodes {
             insertEdge(from: source, to: node, value: 1)
+            // FIXME: insertEdge(from: source, to: node, value: epsilon *
+            // lookupTable[source.pitchclass, source.index, node.pitchclass, node.index])
             insertEdge(from: node, to: sink, value: 1)
+            // FIXME: insertEdge(from: node, to: sink, value: epsilon *
+            // lookupTable[node.pitchclass, node.index, sink.pitchclass, sink.index])
             for other in internalNodes.lazy.filter({ $0 != node }) {
                 insertEdge(from: node, to: other, value: 1)
+                // FIXME: if node.offset == other.offset && node.index == 1 {
```

Comment:
Given this requirement, it might make sense to shove in the pairs of nodes wrapped up together. Then we don't need to ask whether they represent the same item.

Reply:
agreed